.github/workflows/codeql.yml

```
@@ -1,17 +1,101 @@
 # Workflow for advanced CodeQL setup used for scanning Java/JavaScript/TypeScript/Vue/Python based source files
 name: "CodeQL Advanced"
+env:
+      # Whether to analyze Java code or not (only set to true if repo has Java source code)
+      analyze-java: true
+      # Build mode to use for analysis of Java code (e.g. none, autobuild, manual)
+      java-buildmode: "autobuild" 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)